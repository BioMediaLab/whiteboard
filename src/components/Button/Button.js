import { Button } from '@shopify/polaris'

export default Button
