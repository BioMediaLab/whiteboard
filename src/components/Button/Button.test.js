import React from 'react'
import Button from './'

describe('Button', () => {
  test('can render', () => {
    const component = render(<Button type="submit">Save</Button>)

    expect(component).toMatchSnapshot()
  })

  test('can handle click', () => {
    const onClick = jest.fn()
    const component = render(<Button type="submit" onClick={onClick} />)

    component.click()

    expect(onClick).toHaveBeenCalled()
  })
})
