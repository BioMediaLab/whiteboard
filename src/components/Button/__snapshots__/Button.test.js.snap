// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button can render 1`] = `
<div>
  <button
    class="Button"
    type="submit"
  >
    Save
  </button>
</div>
`;
